scroll-search.lic

```diff
@@ -0,0 +1,533 @@
+=begin
+  Documentation: TBD
+=end
+
+custom_require.call(%w(common common-items drinfomon))
+
+# Define Scroll Info class
+class ScrollInfo
+  def initialize(container, index, container_index, ordinal, spell, type)
+	# Instance variables
+	@container = container
+	@index = index
+	@container_index = container_index
+	@ordinal = ordinal
+	@spell = spell
+	@type = type
+  end
+
+  def container
```

Comment:
Ruby has something called `attr_accessor`, which basically is a shorthand for defining a getter and setter. You can see an example in combat-trainer (GameState). That might simplify the ScrollInfo class a bit.